main/modes/picross/mode_picross.c

```diff
@@ -1134,34 +1427,36 @@ void saveCompletedOnSelectedLevel(bool completed)
                 victories = victories & ~(1 << (p->selectedLevel->index));
             }
             //Save new number
-            writeNvs32("picross_Solves0", victories);
+            writeNvs32(picrossCompletedLevelData1, victories);
         }else if(p->selectedLevel->index < 64)//levels 32-64
         {
             victories = 0;
-            readNvs32("picross_Solves1", &victories);
+            readNvs32(picrossCompletedLevelData2, &victories);
             if(completed){
                 victories = victories | (1 << (p->selectedLevel->index - 32));
             }else{
                 victories = victories & ~(1 << (p->selectedLevel->index - 32));
             }
-            writeNvs32("picross_Solves1", victories);
+            writeNvs32(picrossCompletedLevelData2, victories);
         }else if(p->selectedLevel->index < 96)//levels 65-95
         {
             victories = 0;
-            readNvs32("picross_Solves2", &victories);
+            readNvs32(picrossCompletedLevelData3, &victories);
             if(completed){
                 victories = victories | (1 << (p->selectedLevel->index - 64));
             }else{
                 victories = victories & ~(1 << (p->selectedLevel->index - 64));
             }
-            writeNvs32("picross_Solves2", victories);
+            writeNvs32(picrossCompletedLevelData3, victories);
         }
 }
 //I promise I tried to do a proper index->row/col mapping here, since we only need 100*2 bits of data. (<320 used here) but i kept messing the math up.
 //I know that 120 bits is not a lot, but on principle, it bothers me and I would like to optimize this in the future.
+//update: now that I am storing up to 15x15 puzzles (30 bits of data in 32 bits of space, its only 15rows*2 bits of wasted bits. nice.)
 
 char * getBankName(int i)
 {
+    //this is what it is.
```

Comment:
it could be `sprintf(bankName, "pic_b_%d", i);` (or maybe banks aren't necessary at all with blobs)